test-harness/specs/dynamic_json_responses-AC-1.dynamic-response.oas2.txt

```diff
@@ -0,0 +1,39 @@
+====test====
+Prism generates dynamic values for schema when no static examples are provided.
+
+Given there are no examples defined for an operation
+And this operation can produce application/json response
+And this operation response defines a JSON Schema (schema field)
+And I make a request to this operation
+And the request has Accept header of application/json
+Then I should get a response with dynamically generated JSON as payload
+====spec====
+swagger: "2.0"
+info:
+  title: 'title'
+  version: '1.0'
+paths:
+  /todos:
+    get:
+      produces:
+        - application/json
+      responses:
+        200:
+          description: Get Todo Items
+          schema:
+            type: object
+            properties:
+              title:
+                type: string
+              description:
+                type: string
+              priority:
+                type: integer
+====server====
+mock -p 4010
```

Comment:
The dynamic example should be explicitly enabled
```suggestion
mock -d -p 4010
```

This was not catch because of the problem I exposed above: the payload will always pass the schema validation because the schema is generated from the payload.

Reply:
@XVincentX, isn't `dynamic generation` enabled by default?

Reply:
No, by default is not enabled.